commit/merkleroot/outcome.go

```diff
@@ -188,6 +188,7 @@ func buildReport(
 		RootsToReport:       roots,
 		OffRampNextSeqNums:  prevOutcome.OffRampNextSeqNums,
 		RMNReportSignatures: sigs,
+		RMNRemoteCfg:        prevOutcome.RMNRemoteCfg,
```

Comment:
Shouldn't we come to consensus on this prior to including it in the outcome?

Reply:
The consensus is reached in another round where state is `SelectingRangesForReport`. https://github.com/smartcontractkit/chainlink-ccip/blob/febfb02821d8ba19477958cafec609bc1774b1b2/commit/merkleroot/outcome.go#L52

There we will gather all the AO and reach consensus.

Here we are only propagating the RMNRemote config.

Reply:
Oh I see, yeah that makes sense 👍